sdk/apimanagement/Azure.ResourceManager.ApiManagement/src/Customized/Models/ApiCreateOrUpdateContent .cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.ApiManagement.Models
+{
+    /// <summary> Summary of revision metadata. </summary>
+    public partial class ApiCreateOrUpdateContent
+    {
+        /// <summary> A URL to the Terms of Service for the API. MUST be in the format of a URL. </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        [Obsolete("This property has been replaced by TermsOfServiceUriLink", false)]
```

Comment:
we usually should not add `Obsolete`, because removing it is forbidden by ApiCompat. Just in case.